tools/eeprom-tool/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8)
+project(eeprom-tool LANGUAGES CXX C)
+
+set(CMAKE_CXX_STANDARD 11)
+
+add_executable(${PROJECT_NAME} main.cpp eeprom_tool.cpp cam96tof1_eeprom.cpp cli_helper.cpp camera_eeprom_factory.cpp ${GETOPT_C_FILE})
```

Comment:
The ${GETOPT_C_FILE} doesn't seem to be required.

Reply:
fixed